### ISSUE TITLE
[Bug]: Fatal exception whilst installing modpack

### ISSUE THREAD

**Issue by FlufferNutt** [FTBTeam/FTB-App#777]
### What Operating System

Windows 10

### App Version

202211151621-329441ed6d-release

### UI Version

329441ed6d

### Log Files

https://pste.ch/foyaxenanu

### Debug Code

FTB-DBGOYAJIRESES

### Describe the bug

Attempting to install modpacks through the FTB app leads to fatal exceptions

### Steps to reproduce

1. Open FTB app 
2. Attempt to install modpack
3. Process halts with fatal exception

### Expected behaviour

Modpacks install correctly

### Screenshots

![Screenshot 2022-11-30 152649](https://user-images.githubusercontent.com/119545158/204929827-98593227-f8ce-497a-8fd8-7524346edcf0.png)

### Additional information

This issues occurs with all tested modpacks

### Information

- [X] I have provided as much information as possible

**Comment by covers1624** [comment]
Please make sure `api.adoptium.net` is not blocked by any network DNS configuration.

The FTBApp was unable to contact the adoptium API to provision a Java runtime.

**Comment by WhyNotCthulhu** [comment]
I do not believe my ISP allows me to alter my DNS configuration. Shutting off my home firewall didn't seem to help any either.

**Comment by MysticaIl** [comment]
Maybe its a problem in the Computer Ram or smth. Im also having this problem

**Comment by WhyNotCthulhu** [comment]
You might have a RAM issue but I can assure you I do not. This is a network error.

**Comment by Howsyomamanem** [comment]
found the button to upload logs.  
here you go: https://pste.ch/nutorosaqu

**Comment by Howsyomamanem** [comment]
looks like it failed to get the jre... not sure if it does this when you download a pack or when you install the app... but the jre folder is just empty...
...\AppData\Local\.ftba\bin\runtime\jdk-17.0.2+8-jre\bin\

**Comment by Howsyomamanem** [comment]
FIXED!  I noticed the required jre is in a zip file on the ...\AppData\Local.ftba\bin\runtime\ folder.  I manually extracted the zip into the target jre folder that should have been populated.   now the pack install works!

You guys may wanna check your install scripts to make sure this is succeeding or throw an error.  ... or something.

**Comment by covers1624** [comment]
@WhyNotCthulhu @Howsyomamanem Please do not hijack issues.

**Comment by Ragnadom** [comment]
I am also having this issue while trying to install plexiglass 1.2.3.

**Comment by cosmo5001** [comment]
https://www.windowscentral.com/how-change-your-pcs-dns-settings-windows-10
This is a great guide to change your DNS settings on windows, using Google's free DNS server completely fixed the problem for me.

**Comment by gelbytes** [comment]
Update: 1.1.1.1 and 8.8.8.8 both produce the following behavior in Windows ping:

```C:\Users\me>ping api.adoptium.net

Pinging adoptium-api.eastus.cloudapp.azure.com [20.62.244.126] with 32 bytes of data:
Request timed out.
Request timed out.
Request timed out.
Request timed out.

Ping statistics for 20.62.244.126:
    Packets: Sent = 4, Received = 0, Lost = 4 (100% loss),
```

Inexplicably, when I visit the site in Firefox, I get this:
![Firefox Developer Edition, showing a documentation homepage, api.adoptium.net in the URL bar](https://user-images.githubusercontent.com/75345720/210657206-43968144-95d0-4d08-b465-a8a42d2a4393.png)

**Comment by MichaelHillcox** [comment]
Closing as we've had a bunch of updated around networking recently, please update and if you're still having an issue, please open a new issue.
